styles/main/_content.scss

```diff
@@ -286,6 +286,80 @@
 			font-weight: bold;
 		}
 	}
+	// Description ---------------------------------------------------------- //
+	.album_description {
+		width: 100%;
+		margin: 10px 0 0 30px;
+		color: white;
+		font-size: 14px;
+		line-height: 19px;
+		h1,
+		h2,
+		h3 {
+			margin: 20px 0 20px;
+			&:first-child {
+				margin-top: 0;
+			}
+			&:last-child {
+				margin-bottom: 0;
+			}
+		}
+		p,
+		ol,
+		ul,
+		blockquote,
+		pre {
+			margin: 10px 0 10px;
+			&:first-child {
+				margin-top: 0;
+			}
+			&:last-child {
+				margin-bottom: 0;
+			}
+		}
+		h1 {
+			font-size: 24px;
+			line-height: 29px;
+			font-weight: bold;
+		}
+		h2 {
+			font-size: 20px;
+			line-height: 24px;
+			font-weight: bold;
+		}
+		h3 {
+			font-size: 16px;
+			font-weight: bold;
+		}
+		ol {
+			list-style: decimal inside;
```

Comment:
```suggestion
			list-style: decimal outside;
			padding-left: 40px;
```

`ol` and `ul` need some additional settings regarding a hanging indention on the left. Currently, subsequent lines of a multi line bullet point are not properly indented.

Actually, we need a left-padding (e.g. 40px is the default for Firefox) and the position of the bullet point should be within the padding, i.e. outside the principal box of the HTML element. Otherwise visual problems as the following will happen.

![Screenshot_20220116_105542a](https://user-images.githubusercontent.com/7428426/149657633-7df9bd1c-2b38-4900-add3-b9b96d4449e2.png)